src/auto_scheduler/compute_dag.cc

```diff
@@ -276,10 +276,18 @@ std::pair<te::Schedule, Array<te::Tensor>> ComputeDAG::ApplySteps(
     // return value, so the ApplyToSchedule is not able to be merged to single interface
     if (auto ps = step.as<ReorderStepNode>()) {
       ps->ApplyToSchedule(stages, stage_to_axes);
+    } else if (auto ps = step.as<ComputeAtStepNode>()) {
```

Comment:
Why do you use a different order from the order in our internal repo?
I propose we can use two orders
1.  Alphabetical order
2. Logical order from easy to hard.
   - primitives working on one stage: reorder, split, fuse, annotation
    - primitives working on multiple stages: compute_at, compute_root, compute_inline
   - primitive adding new stages: cache_read, cache_write, ...

And we should keep the same order in all places.

Reply:
Ok, in the .h & .py I was tring to place the functions with similar return format together.

void reorder
void compute_at
void compute_root
void compute_inline
Array<Iterator> split
Iterator fuse
Iterator vectorize
Iterator parallel
Iterator unroll
Iterator bind

It's fine to modified to the order you suggested.